core/templates/pages/classroom-page/classroom-page.component.html

```diff
@@ -2,7 +2,12 @@
 <div class="oppia-classroom-viewer-container position-relative">
   <div class="oppia-classroom-header">
     <div class="classroom-header">{{ 'I18N_CLASSROOM_PAGE_HEADING' | translate }}</div>
-    <div class="classroom-name">{{ classroomDisplayName }}</div>
+    <div class="classroom-name" *ngIf="!isClassroomTranslationKeyToBeDisplayed()">
```

Comment:
Same thing, maybe put two ngIf spans inside the div to reduce the repetition. Ditto elsewhere.

Reply:
Done!